src/Arcus.Scripting.LogicApps/Scripts/Cancel-AzLogicAppRuns.ps1

```diff
@@ -0,0 +1,18 @@
+param(
+    [Parameter(Mandatory = $true)][string] $ResourceGroupName = $(throw "Name of the resource group is required"),
+    [Parameter(Mandatory = $true)][string] $LogicAppName = $(throw "Name of the logic app is required")
+)
+
+try{
+    $runs = Get-AzLogicAppRunHistory -ResourceGroupName $ResourceGroupName -Name $LogicAppName | 
+        Where-Object {$_.Status -eq 'Running'}
+
+    foreach ($run in $runs) {
+        Stop-AzLogicAppRun -ResourceGroupName $ResourceGroupName -Name $LogicAppName -RunName $run.Name -Force
+        Write-Verbose "Cancelled run $run.Name for the Azure Logic App '$LogicAppName' in resource group '$ResourceGroupName'"
+    }
+
+    Write-Host "Successfully cancelled all running instances for the Azure Logic App '$LogicAppName' in resource group '$ResourceGroupName'" -ForegroundColor Green
+} catch {
```

Comment:
Should we move the `try/catch` block inside the `foreach` loop so that we don't stop with cancelling Logic Apps when 1 fails? Or does the `Stop-AzLogicAppRun` command not throw an exception on failure and is this `try/catch` block just for our peace of mind?

Reply:
It's more for our peace of mind 😉 
I tried multiple stuff to get it to throw an exception, but next to just entering a wrong Logic App name (in which case it will not enter the foreach) I cannot get it to break.

So if you agree I think we can leave it as is 👍🏻

Reply:
Ok, great. Good that you checked this.
We can indeed leave it like this, then. 😄